src/Raven.Studio/wwwroot/Content/css/main-menu.less

```diff
@@ -313,6 +313,14 @@
                 content: 'M';
             }
         }
+        
+        .giga {
+            background-color: fadeout(@label-default-bg,50%);
+
+            &::before {
+                content: 'G';
```

Comment:
Wait, why G? Those are not bytes, this should be B

Reply:
Based on what? 

If B then K should be T?